cowprotocol/cowswap
# Prepare for Analytics 4 and add settings event

# Summary
 Universal Analytics will be deprecated, and the new Google Analytics 4 will be the official one.

This PR setups the new analytics ID. It makes use of the recent changes from Uniswap which changed the library react-ga (which only supports Universal Analytics) to react-ga4 (which uses almost the same interface and uses ga4)

More info in the deprecation
https://support.google.com/analytics/answer/11583528?hl=en

For this PR it was set up a new analytics configuration for our new organisation, so gnosis one won't be used any more.

This PR depends in the merge in #441 and will be a pre-requirement for the new Funnel that Mindy will prepare.

# To Test
1. Visit the URL in this PR
2. Check we are getting our analytics info in our newly created analytics
